lib/socketio.js

```
@@ -1,7 +1,1274 @@
+      t.on("data", this.onData.bind(this)),
+        t.on("error", t => {
+          this.onError("xhr poll error", t);
+        }),
+        (this.pollXhr = t);
+    }
+  },
+};
+class $ extends u {
+  constructor(t, e = {}) {
+    super(),
+      t && "object" == typeof t && ((e = t), (t = null)),
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)